rskj-core/src/test/java/co/rsk/peg/federation/FederationChangeIT.java

```diff
@@ -283,19 +299,114 @@ private void voteToCommitPendingFederation() {
-        proposedFederation = federationStorageProvider.getProposedFederation(BRIDGE_CONSTANTS.getFederationConstants(), ACTIVATIONS);
-        assertTrue(proposedFederation.isEmpty());
+        // Next call to update collections will create svp fund tx
+        bridgeSupport.updateCollections(UPDATE_COLLECTIONS_TX);
+        bridgeSupport.save();
+
+        var svpFundTxHashUnsigned = bridgeStorageProvider.getSvpFundTxHashUnsigned();
+        assertTrue(svpFundTxHashUnsigned.isPresent());
+        assertEquals(activeFederationUtxosSizeBeforeCreatingFundTx - 1, federationSupport.getActiveFederationBtcUTXOs().size());
+    }
+   
+    private void registerSignedSvpFundTx() throws Exception {
+        var pegoutsTxs = bridgeStorageProvider.getPegoutsWaitingForConfirmations()
+            .getEntries().stream().toList();
+        assertEquals(1, pegoutsTxs.size());
+        var svpFundTx = new BtcTransaction(
+            BRIDGE_CONSTANTS.getBtcParams(), pegoutsTxs.get(0).getBtcTransaction().bitcoinSerialize());
+       
+        signInputs(svpFundTx, ORIGINAL_FEDERATION_MEMBERS_KEYS.subList(0, 5));
+
+        var pmtWithTransactions = 
+            BridgeSupportTestUtil.createValidPmtForTransactions(
+                List.of(svpFundTx.getHash()), BRIDGE_CONSTANTS.getBtcParams());
+        var btcBlockWithPmtHeight =
+            BRIDGE_CONSTANTS.getBtcHeightWhenPegoutTxIndexActivates() + BRIDGE_CONSTANTS.getPegoutTxIndexGracePeriodInBtcBlocks();
+        var chainHeight = btcBlockWithPmtHeight + BRIDGE_CONSTANTS.getBtc2RskMinimumAcceptableConfirmations();
+        BridgeSupportTestUtil.recreateChainFromPmt(
+            btcBlockStore, chainHeight, pmtWithTransactions, btcBlockWithPmtHeight, BRIDGE_CONSTANTS.getBtcParams());
+        bridgeStorageProvider.save();
+
+        int activeFederationUtxosSizeBeforeRegisteringTx = federationSupport.getActiveFederationBtcUTXOs().size();
+        bridgeSupport.registerBtcTransaction(
+            buildRskTx(),
+            svpFundTx.bitcoinSerialize(),
+            btcBlockWithPmtHeight,
+            pmtWithTransactions.bitcoinSerialize()
+        );
+        bridgeSupport.save();
+
+        assertEquals(activeFederationUtxosSizeBeforeRegisteringTx + 1, federationSupport.getActiveFederationBtcUTXOs().size());
+        var svpFundTransactionSigned = bridgeStorageProvider.getSvpFundTxSigned();
+        assertTrue(svpFundTransactionSigned.isPresent());
+        
+        // simulate removal to leave state clean
+        assertTrue(bridgeStorageProvider.getPegoutsWaitingForConfirmations().removeEntry(pegoutsTxs.get(0)));
+    }
+
+    private void createSvpSpendTx() throws Exception {
+        // Next call to update collections will create svp spend tx
+        bridgeSupport.updateCollections(UPDATE_COLLECTIONS_TX);
+        bridgeSupport.save();
+
+        var svpFundTxHashUnsigned = bridgeStorageProvider.getSvpFundTxHashUnsigned();
+        assertFalse(svpFundTxHashUnsigned.isPresent());
```

Comment:
```suggestion
        var svpFundTxSigned = bridgeStorageProvider.getSvpFundTxSigned();
        assertFalse(svpFundTxSigned.isPresent());
```

here you should check the svp fund tx signed value was removed

Reply:
Thanks, updated